hearvalidator/validate.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+"""
+This command-line script can be called to validate any module against the HEAR 2021
+API to ensure that all the current functions and attributes are available, can be called
+with the expected input, and produce correctly formed output. To see the API that this
+was built against, please visit
+https://neuralaudio.ai/hear2021-holistic-evaluation-of-audio-representations.html#common-api
+
+Usage:
+    python3 -m hear-validator <module-to-test> -m <path-to-model-checkpoint-file>
+
+Example usage:
+    python -m hear_validator hearkit.baseline
+
+TODO:
+    - Build this out to support TensorFlow models as well.
+"""
+
+import sys
+import argparse
+import importlib
+import warnings
+
+import torch
+import tensorflow as tf
+
+
+class ModelError(BaseException):
+    """Class for errors in models"""
+
+    pass
+
+
+class ValidateModel:
+
+    ACCEPTABLE_SAMPLE_RATE = [16000, 22050, 44100, 48000]
+
+    def __init__(self, module_name: str, model_file_path: str):
+        self.module_name = module_name
+        self.model_file_path = model_file_path
+        self.device = "cuda:0" if torch.cuda.is_available() else "cpu"
```

Comment:
Pranay made a comment to include this as a command line arg

Reply:
Could use click or fire for that

Reply:
And if no option is passed use this as the default